frameworks/hdfs/tests/test_sanity.py

```diff
@@ -398,3 +398,16 @@ def test_permanently_replace_journalnodes():
         sdk_recovery.check_permanent_recovery(
             config.PACKAGE_NAME, foldered_name, pod, recovery_timeout_s=25 * 60
         )
+
+
+@pytest.mark.sanity
+@pytest.mark.recovery
+def test_namenodes_acheive_quorum_after_journalnode_replace():
+    """
+    This test aims to check that namenodes recover after a journalnode failure.
+    It checks the fix to this issue works: https://jira.apache.org/jira/browse/HDFS-10659
+    """
+    pod_list = ["journal-0", "journal-1", "journal-2"]
+    for pod in pod_list:
+        sdk_recovery.check_permanent_recovery(config.PACKAGE_NAME, foldered_name, pod, recovery_timeout_s=25 * 60,
```

Comment:
LGTM, thanks @soumasish. Let's wait for CI tests to complete successfully

Reply:
Sorry @soumasish I don't think this test does what you think it does. It could be a bug in the `sdk_recovery` tooling but if you search `0510d1b2-78cf-48e3-b01d-db8e35ada60` in the full build log, you'll see that for every pod you replace, eg `journal-0`, the replaced tasks all get appended with the same ID and what ends up happening is the test checks all the new tasks should have new IDs but... it'll always be true bc they had the wrong IDs to start with.

And isn't this a test for namenode finding quorum ie not failing to find the `paxos` directory? Should we not instead check that namenodes are healthy (not replaced) after JN failures _and_ there was no exception thrown in the logs regarding missing paxos?